packages/apollo-links/src/authHttpLink.ts

```diff
@@ -24,6 +24,7 @@ interface BaseAuthOptions {
   fallbackServiceUrl?: string; // fall back service url for `AuthLink`
   scoreStore?: IStore; // pass store in, so it doesn't get lost between page refresh
   maxRetries?: number;
+  useFallbackImmediateAfterGraphqlError?: boolean;
```

Comment:
To make this optional field name a bit short, recommend: `useImmediateFallbackOnError `